glean-core/csharp/Glean/GleanParser.cs

```diff
@@ -0,0 +1,270 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Security;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace GleanTasks
+{
+    public class GleanParser : ToolTask
+    {
+        // The name of the directory in which the Python virtual
+        // environment must be created.
+        private const string DefaultVirtualEnvDir = ".venv";
+
+        // The glean_parser pypi package version 
+        private const string GleanParserVersion = "1.28.3";
```

Comment:
Might want to consider coordinating with #1116 to ensure this gets updated by the script there.

Reply:
Good point! @mdboom how should we manage this? :) I'm happy to have #1116 merge first and then change it a bit, but I'm not super expert with sed tbh.

Reply:
Let's land #1116 first and then update that script in this PR. I'm not much of a sed expert either, I just copied what @badboy did elsewhere.